sql/src/main/java/io/crate/analyze/AlterTableAnalyzer.java

```diff
@@ -88,7 +88,7 @@ private static void maybeRaiseBlockedException(DocTableInfo tableInfo, Settings
     }
 
     @Nullable
-    private static PartitionName getPartitionName(List<Assignment> partitionsProperties,
+    public static PartitionName getPartitionName(List<Assignment> partitionsProperties,
```

Comment:
would now make sense to rename it to createPartitionName in order to distinguish from a simple getter now that it is public. you could also add javadoc do it